x-pack/plugins/reporting/server/browsers/index.ts

```diff
@@ -5,30 +5,21 @@
  * 2.0.
  */
 
-import { first } from 'rxjs/operators';
 import { ReportingCore } from '../';
+import { ReportingConfigType } from '../config';
 import { LevelLogger } from '../lib';
-import { chromium, ChromiumArchivePaths } from './chromium';
+import { ChromiumArchivePaths } from './chromium';
 import { HeadlessChromiumDriverFactory } from './chromium/driver_factory';
-import { installBrowser } from './install';
 
 export { chromium } from './chromium';
 export { HeadlessChromiumDriver } from './chromium/driver';
 export { HeadlessChromiumDriverFactory } from './chromium/driver_factory';
 
-type CreateDriverFactory = (
-  core: ReportingCore,
-  binaryPath: string,
-  logger: LevelLogger
-) => HeadlessChromiumDriverFactory;
-
 export interface BrowserDownload {
-  createDriverFactory: CreateDriverFactory;
+  createDriverFactory: (
+    core: ReportingCore,
+    captureConfig: ReportingConfigType['capture'],
+    logger: LevelLogger
+  ) => HeadlessChromiumDriverFactory;
   paths: ChromiumArchivePaths;
 }
-
-export const initializeBrowserDriverFactory = async (core: ReportingCore, logger: LevelLogger) => {
```

Comment:
`chromium.createDriverFactory` is a one-liner now, so this helper function isn't helpful :)